Packs/GroupIB_ThreatIntelligenceAttribution/IndicatorFields/indicatorfield-GIB_Admiralty_Code.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
@EvgeniyMeteliza Please add the "associatedTypes" key to all of the indicator fields with the relevant indicator types as a value (as you did with the incident fields).

Reply:
Hi @EvgeniyMeteliza, I believe that this is something we would want to avoid, please provide a different name for each indicator type.

Reply:
Hi,

It is not my types, unfortunately. They were there from the beginning, so I suppose they are the system ones.

Reply:
That's why I have concerns about adding associated types to every indicator field.

Reply:
But if this behavior is incorrect I can reinstall Demisto instance, because right know I also have huge problems with it, it become laggy and many things just don't work

Reply:
@EvgeniyMeteliza I sent you a DM in DFIR slack workspace - I would like to take a look at these indicators types if possible before suggesting anything.

Reply:
@EvgeniyMeteliza As discussed in the demo, please associate the relevant indicator types to your incident fields and this key as well: ` "associatedToAll": false`
Also, please update here if the duplicate indicator types are still there after an upgrade.